test/jdk/security/infra/java/security/cert/CertPathValidator/certification/CAInterop.java

```diff
@@ -632,6 +654,12 @@ private CATestURLs getTestURLs(String alias) {
             case "certainlyroote1":
                     return new CATestURLs("https://valid.root-e1.certainly.com",
                             "https://revoked.root-e1.certainly.com");
+            case "globalsignr46" ->
+                    new CATestURLs("https://valid.r46.roots.globalsign.com",
+                            "https://revoked.r46.roots.globalsign.com");
```

Comment:
Does this compile? Switch expressions are JDK 14+.

Reply:
It did compile for me... maybe I compiled it with the wrong JDK!

Reply:
Yes. But unless you run the test it won't get compiled.

Reply:
```
make run-test TEST="test/jdk/sun/security/lib/cacerts/VerifyCACerts.java test/jdk/security/infra/java/security/cert/CertPathValidator"
```

Reply:
In any case, the code is changed.

Reply:
I also had `--disable-warnings-as-errors` as a flag while building. Maybe that was it.

Reply:
That was it, it won't happen again :)

After removing the ` --disable-warnings-as-errors` flag, it throws an error:

```
/home/delawen/git/jdk11u-dev/test/jdk/security/infra/java/security/cert/CertPathValidator/certification/CAInterop.java:664: error: illegal start of expression
            case "globalsignr46" ->
                                 ^
```